templates/taxi/car_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete Car</h1>
+  <p>Are you sure you want to delete {{ car }}?</p>
+  <p>All drivers with this car will be also deleted</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <button type="submit" class="btn btn-danger">Delete</button>

```

Comment:
check `checklist.md`  cancel button will be more user-friendly in this case:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.